makefile

```diff
@@ -20,6 +20,8 @@ ci: gen-java gen-swagger
 .PHONY: gen-go
 gen-go:
 	$(foreach file,$(PROTO_FILES),$(call exec-command,protoc --go_out=plugins=grpc:$(GOPATH)/src $(file)))
+	protoc --grpc-gateway_out=logtostderr=true,grpc_api_configuration=opentelemetry/proto/collector/trace/v1/trace_service_http.yaml:$(GOPATH)/src opentelemetry/proto/collector/trace/v1/trace_service.proto
+	protoc --grpc-gateway_out=logtostderr=true,grpc_api_configuration=opentelemetry/proto/collector/trace/v1/trace_service_http.yaml:$(GOPATH)/src opentelemetry/proto/collector/metrics/v1/metrics_service.proto
```

Comment:
This is not a typo, `trace_service_http.yaml` contains mappings for both tracing and metrics for some reason.

Reply:
Fixed, moved metrics part of the config to appropriate dir.